pkg/extender/extender_test.go

```diff
@@ -1881,6 +1911,66 @@ func multiVolume3PreferRemoteOnlyAntiHyperConvergenceTest(t *testing.T) {
 		prioritizeResponse)
 }
 
+// Deploy both anti hyperconveged volumes with preferRemoteNode false and true respectively together with regular volumes
+// Verify hyperconvergence for AntiHyperConvergence volume with preferRemoteNode parameter set to false
```

Comment:
nit.. Let's remove "AntiHyperConvergence" from this comment

Reply:
Will rename it to NeedsAntiHyperConvergence volume